WorkspaceServer/Packaging/PackageBase.cs

```diff
@@ -114,6 +114,7 @@ protected async Task DotnetBuild()
                     File.WriteAllText(
                         LastBuildErrorLogFile.FullName,
                         string.Join(Environment.NewLine, result.Error));
+                    throw new Exception($"Build didn't pass->{result}");
```

Comment:
Avoid throwing `Exception` because catching it requires catching `Exception` and it's always best to catch only the specific exception types you know how to handle.

We have a `CommandLineInvocationException` that you can use for this.